tests/swin/test_modeling_swin.py

```diff
@@ -261,14 +263,15 @@ def test_attention_outputs(self):
             with torch.no_grad():
                 outputs = model(**self._prepare_for_class(inputs_dict, model_class))
             attentions = outputs.encoder_attentions if config.is_encoder_decoder else outputs.attentions
-            self.assertEqual(len(attentions), len(self.model_tester.depths))
+            self.assertEqual(len(attentions), expected_num_attentions)
 
             if chunk_length is not None:
                 self.assertListEqual(
                     list(attentions[0].shape[-4:]),
                     [self.model_tester.num_heads[0], window_size_squared, chunk_length, window_size_squared],
                 )
             else:
+                # attentions is a tuple of tuple, since we have one attention per layer
                 self.assertListEqual(
                     list(attentions[0].shape[-3:]),
                     [self.model_tester.num_heads[0], window_size_squared, window_size_squared],
```

Comment:
You can remove the "chunk length" stuff since Swin doesn't have it.

Also, this comment doesn't seem correct to me, attentions is not a tuple of tuple right?